express/server.js

```diff
@@ -188,6 +213,18 @@ app.post('/checkout', async(req, res) => {
 
 });
 
+app.get('/api', (req, res) => {
```

Comment:
Generic sounding endpoint name. Many api's are prefixed with "/api" so it's good.

We'll rename endpoints for better Story lines on a future PR. First let's make sure all this new stuff is working and we comprehend it, it fits our needs.